docs/Guide/getting-started/creating-a-basic-app-command.mdx

```diff
@@ -38,9 +38,13 @@ through the slash command.
 In order for your bot to respond to the slash command when it is invoked by a user, you must implement the
 `chatInputRun` method for the `Command` class.
 
+```bash npm2yarn2pnpm
+npm install @sapphire/discord.js-utilities
+```
```

Comment:
```suggestion
```

`@sapphire/discord.js-utilities` is a transitive dependency of `@sapphire/framework` so you don't explicitly need to install it really.